src/dali_executor/CMakeLists.txt

```diff
@@ -50,8 +50,8 @@ add_custom_command(
 if(${TRITON_DALI_SKIP_DOWNLOAD})
     get_dali_paths(DALI_INCLUDE_DIR DALI_LIB_DIR DALI_LIBRARIES)
 else()
-    set(DALI_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/dalienv/lib/python3.8/site-packages/nvidia/dali/include)
-    set(DALI_LIB_DIR ${CMAKE_CURRENT_BINARY_DIR}/dalienv/lib/python3.8/site-packages/nvidia/dali)
+    set(DALI_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/dalienv/lib/python${PYTHON_VERSION}/site-packages/nvidia/dali/include)
```

Comment:
Please add `update` i L45 to make sure you get the latest python.

Reply:
Should I be pushing for the latest Python? This PR aims at specifying explicitly Python version in the CMake. Therefore I believe `update` there won't be aligned with the purpose of this PR

Reply:
But you may end up with an old env (although I'm not 100% sure) that may raise a flag during the venerability scan.
Just a suggestion.